career/interviewing/technical-interviewing.md

```diff
@@ -57,9 +57,9 @@ Walk through several example questions and demonstrate successful and unsuccessf
 Look through the materials sections of this guide and try a variety of different challenges. Do several questions for each website and then move on to the next one. Get comfortable with questions that feel and look unfamiliar.
 
 ### Challenge
-- Build a small React app that fetches some data from an API. Try finding something at [Any API](https://any-api.com/?tag=open%20data) for example to provide an API service to use.
-- When the participant is able to fetch data and render it in their own React app have them manipulate the data in some way. For example, make every third word of a paragraph all caps.
-
+- Write a Binary Search program using any programming language.
+- Print all permutations of String both iterative and Recursive way.
```

Comment:
are you sure about the capitalization in this line?

Reply:
@alodahl  I think this is better.
```suggestion
- Print all permutations of string in both the iterative and recursive way.
```